.travis.yml

```diff
@@ -37,6 +36,14 @@ branches:
 before_install:
   - date --rfc-3339=seconds
   - nvm install
+  - pkill -9 -f elasticsearch || true
+  - curl -s -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.5.2-amd64.deb
```

Comment:
We have to download this manually to get the version we want otherwise it defaults to ES 2.3